client/task-list/tasks/PaymentGatewaySuggestions/index.js

```diff
@@ -129,15 +123,18 @@ export const PaymentGatewaySuggestions = ( { query } ) => {
 		} );
 	}, [] );
 
-	const recommendation = useMemo( () => {
-		for ( const id in RECOMMENDED_GATEWAY_IDS ) {
-			const gateway = paymentGateways.get( id );
-			if ( gateway ) {
-				return gateway;
-			}
-		}
-		return null;
-	}, [ paymentGateways ] );
+	const recommendation = useMemo(
```

Comment:
Does not need to block this PR, but this function might be good to separate out for testing purposes.